### PR TITLE
kubectl: add page

### PR DESCRIPTION
**Pull request by serragnoli**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/kubectl.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- List all pods with more information (such as node name):
+
+`kubectl get pods -o wide`
+
+- List all resources with different types:
+
+`kubectl get all`
+
+- Update specified pod with the label 'unhealthy' and the value 'true':
```
Comment (agnivade) [inline pages/common/kubectl.md]: Move this one example up.

File: pages/common/kubectl.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`kubectl label pods {name} unhealthy=true`
```
Comment (agnivade) [inline pages/common/kubectl.md]: {name} -> {{name}}

File: pages/common/kubectl.md (hunk 1 above)
Comment (serragnoli) [reply pages/common/kubectl.md]: Done

File: pages/common/kubectl.md (hunk 2 above)
Comment (serragnoli) [reply pages/common/kubectl.md]: Done

**Pull request merged**